### PR TITLE
Contribution

### PR DESCRIPTION
**Pull request by chuckie82**
1) Updated reference: Nikea to scikit-xray
2) Instructions on cloning using https option
3) Instructions for fetching upstream

### CODE REVIEW

File: contributing.txt [hunk 1]
```diff
@@ -7,20 +7,30 @@ Here's the long and short of it:
...
       git clone git@github.com:your-username/scikit-xray.git
+      or
+      git clone https://github.com:your-username/scikit-xray.git
 
    * Add the upstream repository::
 
-      git remote add upstream git@github.com:Nikea/scikit-xray.git
+      git remote add upstream git@github.com:scikit-xray/scikit-xray.git
+      or
+      git remote add upstream https://github.com:scikit-xray/scikit-xray.git
```
Comment (ericdill) [inline contributing.txt]: I'm pretty sure that needs to be a `/` between `github.com` and `scikit-xray`

File: contributing.txt [hunk 2]
```diff
@@ -7,20 +7,30 @@ Here's the long and short of it:
...
+
+   * Fetch the upstream repository::
+
+      git fetch upstream
 
    * Now, you have remote repositories named:
 
+      git branch -a
```
Comment (ericdill) [inline contributing.txt]: this should be `git remote -v` given the output following this command

**Comment by ericdill** [comment]
Looks good other than a few minor line comments!  Thanks for doing this!

**Comment by chuckie82** [comment]
I made the changes.

File: contributing.txt [hunk 3]
```diff
@@ -7,20 +7,30 @@ Here's the long and short of it:
...
-   * Go to `https://github.com/Nikea/scikit-xray
-     <http://github.com/Nikea/scikit-xray>`_ and click the
+   * Go to `https://github.com/scikit-xray/scikit-xray
+     <http://github.com/scikit-xray/scikit-xray>`_ and click the
      "fork" button to create your own copy of the project.
 
    * Clone the project to your local computer::
 
       git clone git@github.com:your-username/scikit-xray.git
+      or
```
Comment (tacaswell) [inline contributing.txt]: To render right in sphinx these need to be

```
computer ::

   git clone...

or ::
   
    git clone ..

```

**Comment by chuckie82** [comment]
I added 2 double colons

File: contributing.txt (hunk 3 above)
Comment (ericdill) [inline contributing.txt]: I didn't think that this was a sphinx document

Comment (ericdill) [inline contributing.txt]: Are we rendering it as sphinx somewhere?

**Comment by ericdill** [comment]
Looks good, thanks for updating!

**Pull request merged**